gateway/middleware.go

```diff
@@ -407,6 +407,14 @@ func (t BaseMiddleware) ApplyPolicies(session *user.SessionState) error {
 							}
 						}
 
+						for _, t := range v.AllowedTypes {
```

Comment:
Is it possible we could improve the naming of these iterators? I don't see a reason why they couldn't be explicitly clear what the represent.

Reply:
Golang authors promote short variable names https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
I think I have never disagreed with something more in life. Rather than looking at it and instantly knowing what means what, I have to keep looking back and forth to remind myself what this letter means or what that letter means. I understand that this isn't necessarily your philosophy, but it goes against every convention of Uncle Bob's Clean Code.